### ISSUE TITLE
sometimes the player seems to not initialize correctly and buffers indefinitely

### ISSUE THREAD

**Issue by LuisKleinwort** [PierfrancescoSoffritti/android-youtube-player#851]
I use one ore more instances of the player in a Fragment, and after repeatedly opening and closing the section where said Fragment is used, the player buffers forever in some cases.
In my testing I found that when the issue occurs none of the callbacks are called (in this case I'm using getYouTubePlayerWhenReady)

```
doOnAttach {
  findViewTreeLifecycleOwner()!!.lifecycle.addObserver(youtube_player)
  
  youtube_player.getYouTubePlayerWhenReady(object : YouTubePlayerCallback {
    override fun onYouTubePlayer(youTubePlayer: YouTubePlayer) {
      onItemPresentedAction()
      youTubePlayer.cueVideo(videoId, 0F)
    }
  })
}
```

**Comment by PierfrancescoSoffritti** [comment]
Hi, probably duplicate of #840 and/or #830

**Comment by chrisnkrueger** [comment]
Seems to be the same problem. It also happens when the YouTube player is in an Activity.

**Comment by PierfrancescoSoffritti** [comment]
This should be fixed in version 11.1.0.
